database/migrations/2021_09_12_220452_add_solution_selector_id_to_threads_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddSolutionSelectorIdToThreadsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('threads', function (Blueprint $table) {
+            $table->integer('solution_selector_id')->unsigned()->nullable()->default(null);
```

Comment:
I'd like to explore a different naming for this: `answered_by` which is a bit shorter and more readable. `selector` sounds a bit weird.